packages/sns/src/sns.wrapper.ts

```diff
@@ -108,6 +108,9 @@ export class SnsWrapper {
     params: Omit<SnsListNeuronsParams, "certified">
   ): Promise<Neuron[]> => this.governance.listNeurons(this.mergeParams(params));
 
+  listNervousSystemFunctions = () =>
```

Comment:
- could you add `... = (params: Omit<QueryParams, "certified">) => ...` to follow the other function pattern?

- could you type the return type?

Reply:
or it's `Promise<void>`?

Reply:
Sure

Reply:
done